src/applications/disability-benefits/all-claims/tests/content/toxicExposure.unit.spec.jsx

```diff
@@ -317,4 +337,173 @@ describe('toxicExposure', () => {
       expect(errors.toxicExposureConditions.addError.called).to.be.false;
     });
   });
+
+  describe('dateRangePageDescription', () => {
+    it('displays description when counts specified', () => {
+      const tree = render(dateRangePageDescription(1, 5, 'Egypt'));
+      tree.getByText('1 of 5: Egypt', { exact: false });
+      tree.getByText(dateHelp);
+    });
+
+    it('displays description when counts not specified', () => {
+      const tree = render(dateRangePageDescription(0, -1, 'Egypt'));
```

Comment:
I don't think the `totalItems` value will ever be negative? And as I commented within that function, we're gonna see `1 of 1: Egypt` in the case where `dateRangePageDescription(1, 1, 'Egypt')` is used